### DISCUSSION TITLE
Updating repository data failed

### DISCUSSION THREAD

**Discussion by Jeeva2011** [ezra-bible-app/ezra-bible-app#941] [General]
While configuration of bibles, I got message "UPDATING RESPOSITORY DATA FAILED! ARE YOU OFFLINE " I tried many times in Windows,  Android also.
Please help to install it in Android

**Comment by tobias-klein** [comment]
This may be caused by a slow internet connection or a long geographical distance to the servers.
May I know which location you are based in?

Some time ago I [described a workaround](https://github.com/ezra-bible-app/ezra-bible-app/issues/155#issuecomment-750302401). That is something you could try on Windows (not easily possible on Android, though).
